lab_3/start.py

```diff
@@ -13,6 +15,38 @@
     SECRET_SAMPLE = """ Некој е болен и тој не е слободен. Dлетува гол во дупка од мраз. 
     И пее, а плаче од болка. Дали е ова контраст, можеби – живот?"""
 
+    def score_6():
```

Comment:
this function should be written out of __name__=="__main__" condition and the it should be just called inside it as you did on the 48th line